trtools/utils/tr_harmonizer.py

```diff
@@ -680,6 +682,15 @@ def __init__(self,
             self.has_fabricated_ref_allele = False
             self.ref_allele_length = len(ref_allele) / len(motif)
 
+        # declaration of end_pos variables
+        self.end_pos = int(self.pos + self.ref_allele_length * len(motif) - 1)
```

Comment:
This cast is unsafe, `int(1.9999)` goes to 1, and it's possible that with finite precision this doesn't properly evaluate to an exact integer.  Let's use round instead. We can have a comment here explaining why we're using round when all we want is a cast because in theory it should be a whole integer.